src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +17,7 @@
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<T> getAll(Class<? extends T> type);
 
     /**
```

Comment:
Don't begin class or method implementation with empty lines. You should remove the unnecessary empty lines before the `fill` method declaration.